content/reference/cli/supported-commands.md

```diff
@@ -0,0 +1,99 @@
+| `merge`    | ✅         |                                                             |
+| `log`      | 🟠        | planned support for `dolt log --all` and `dolt log --graph` |
+| `stash`    | ❌         |                                                             |
+| `tag`      | ✅         |                                                             |
+| `worktree` | ❌         |                                                             |
+
+## Sharing and Updating Projects
+
+| Component | Supported | Notes and limitations |
+|:----------|:----------|:----------------------|
+| `fetch`   | ✅         |                       |
+| `pull`    | ✅         |                       |
+| `push`    | ✅         |                       |
+| `remote`  | ✅         |                       |
+
+## Inspection and Comparison
+
+| Component    | Supported | Notes and limitations                               |
+|:-------------|:----------|:----------------------------------------------------|
+| `show`       | 🟠        | not supported for merge commits or internal objects |
+| `diff`       | ✅         |                                                     |
+| `range-diff` | ❌         |                                                     |
+| `shortlog`   | ❌         |                                                     |
+| `describe`   | ❌         |                                                     |
+
+## Patching
+
+| Component     | Supported | Notes and limitations |
+|:--------------|:----------|:----------------------|
+| `cherry-pick` | ✅         |                       |
+| `rebase`      | ❌         |                       |
+| `revert`      | ✅         |                       |
+
+## Debugging
+
+| Component | Supported | Notes and limitations |
+|:----------|:----------|:----------------------|
+| `bisect`  | ❌         | support is planned    |
+| `blame`   | ✅         |                       |
+
+## Administration
+
+| Component       | Supported | Notes and limitations                  |
+|:----------------|:----------|:---------------------------------------|
+| `clean`         | 🟠        | does not completely match git behavior |
+| `gc`            | ✅         |                                        |
+| `fsck`          | ❌         |                                        |
+| `reflog`        | ❌         | support is planned                     |
+| `filter-branch` | ✅         |                                        |
+
+## Plumbing Commands
+
+| Component      | Supported | Notes and limitations |
+|:---------------|:----------|:----------------------|
+| `check-ignore` | ❌         |                       |
+| `ls`           | ✅         |                       |
+| `merge-base`   | ✅         |                       |
+| `rev-list`     | ❌         |                       |
+| `rev-parse`    | ❌         |                       |
+| `name-rev`     | ❌         | support is planned    |
```

Comment:
the "support is planned" phrase implies all the other things aren't planned, which isn't really the case. Seems like you should drop this phrase and somewhere else in that page encourage users to contact us if there is something they want to see prioritized.

The risk is that these will quickly fall out of date as priorities change.